gems/awesome-bot/CVE-2019-15224.yml

```diff
@@ -0,0 +1,21 @@
+---
+gem: awesome-bot
+url: https://nvd.nist.gov/vuln/detail/CVE-2019-15224
+date: 2019-08-20
+title: Critical severity vulnerability that affects rest-client, cron_parser, coin_base,
```

Comment:
Can we describe what's happening here? and might as well make this specific to the gem in question.

Reply:
done.